### DISCUSSION TITLE
Do You Support the commercial templates from Joomdev ?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#137] [General]
I ask because a web dev ask this on my Post I create on Joomla forum to announce that **Astroid was well & alive** 
—> https://forum.joomla.org/viewtopic.php?f=713&t=990697&p=3661437#p3661437
1671 views

Do You Support the commercial templates from Joomdev ?

**Comment by sonvnn** [comment]
@Chacapamac Thank you very much for your effort with Astroid. I appreciate that.
For Joomdev Templates: Stackideas will be taking over the commercial templates from Joomdev. You can pm to @imarklee to ask about them or contact them on https://stackideas.com/ . They will answer all question about them.

Kind Regards,
Sonny

**Comment by Chacapamac** [comment]
Great Stuff, I will ask and come back here.
If StackIdeas, take over the support for the templates, do you think it will be a good idea to link people and developers that need that support on in the New Astroid website. They will need to update their Joomdev templates to the New Astroid anyway.
What do you think?

**Reply by sonvnn** [reply]
Yeah, I very sure that Stackideas will support Astroid here. Because, Mark Lee (@imarklee CEO of Stackideas) is one of Admins in this Git. His team has joined to this git and will help us improve and fix issues related Astroid. They will update Joomdev templates to new Astroid, too. I sure that.

**Reply by Chacapamac** [reply]
Thanks @sonvnn — I put the announcement on my post about Astroid in the Joomla forum —> https://forum.joomla.org/viewtopic.php?f=713&t=990697

I think we should add  some direct link to StackIdeas to redirect people that need that support in the website astroidframe.work

Personally I was thinking in first adding a simple 3 actions buttons/zones in a support/Template section on the frontpage that redirect to the right places
Support/Templates section
———————————————————————————————————
**Support & Astroid Templates**
Support for Astroid Framework     |     Support for Joomdev Commercials Templates      |     NEW Astroid Templates            
———————————————————————————————————

**Support for Astroid Framework** —> Go to https://github.com/templaza/astroid-framework/discussions

**Support for Joomdev Commercials Templates** —> Go to a directly to StackIdeas Joomdev Templates support link
— Also need to be added under the “Resources” main menu item

**NEW Astroid Templates** —> Go to https://astroidframe.work/resources/template-providers


Personally, I will change the main menu “**Resources**” to “**Resources & Templates**” or even better Just “**Templates**” if you don’t think any other type of resource will appear in that menu. 

**People need to know right away, in the front page, that this is a full service website around the NEW Astroid.**

**Comment by Chacapamac** [comment]
Thanks @sonvnn , other users ask when they can expect to have a support for the templates from StacksIdeas and if they also will take care of the Joomdev Builder and other extensions?
